source/column_resizer.ts

```diff
@@ -90,25 +102,39 @@ export class ColumnResizer {
 
   private s2() {
     this.state = 2;
+    this.table.showResizeCursor();
   }
 
-  private s3(event: MouseEvent) {
+  private s3() {
     this.state = 3;
-    this.table.onResize(this.currentIndex, event.movementX);
-    return this.s2();
+  }
+
+  private s4(event: PointerEvent) {
+    this.state = 4;
+    const rectangle = this.table.getColumnRect(this.currentIndex);
+    if(event.clientX > rectangle.left) {
+      this.table.onResize(
+        this.currentIndex, 
+        Math.abs(event.clientX - rectangle.left));
+    }
+    return this.s3();
   }
 
   private getLabel(point: {x: number, y: number}) {
     let label = -1;
-    if(this.table.corners.topLeft.y <= point.y &&
-        this.table.corners.bottomRight.y >= point.y &&
-        this.table.corners.topLeft.x <= point.x &&
-        this.table.corners.bottomRight.x >= point.x) {
-      let edge = this.table.corners.topLeft.x;
-      for(let i = 0; i < this.table.columnCount; ++i) {
-        edge += this.table.getColumnWidth(i);
-        const innerEdge = edge - this.table.activeWidth;
-        if(innerEdge <= point.x && point.x <= edge ) {
+    for(let i = 0; i < this.table.columnCount; ++i) {
+      const leftRectangle = this.table.getColumnRect(i);
```

Comment:
What is "left" about the rectangle?

Reply:
It's to the left of the boundary/divider.